quickstart/daml/licensing-tests/daml.yaml

```diff
@@ -11,12 +11,12 @@ dependencies:
   - daml3-script
 data-dependencies:
   # TODO: fetch as artifacts when integrating gradle-daml-plugin https://github.com/DACH-NY/canton-network-node/issues/16393
-- ../dars/splice-amulet-0.1.6.dar
-- ../dars/splice-util-0.1.1.dar
-- ../dars/splice-wallet-payments-0.1.6.dar
-- ../dars/splice-amulet-test-0.1.6.dar
-- ../dars/splice-wallet-test-0.1.6.dar
+- ../dars/splice-amulet-0.1.8.dar
+- ../dars/splice-wallet-payments-0.1.8.dar
+- ../dars/splice-amulet-test-0.1.8.dar
+- ../dars/splice-wallet-test-0.1.8.dar
```

Comment:
Let's add a README.md to the `/dars/` directory explaining from what commit the .dars were copied, or how they were built.